extensions/dacpac/src/wizard/pages/deployPlanPage.ts

```diff
@@ -118,6 +120,14 @@ export class DeployPlanPage extends DacFxConfigPage {
 				label: loc.proceedDataLossMessage,
 			}).component();
 
+		this.dataLossCheckbox.onChanged(() => {
```

Comment:
Instead of capturing every time a user clicks why not just send a summary of selected options you care about at the end? That'll avoid needless spam if users click on the checkboxes a whole bunch (unless you think you'd care about such a scenario)

Reply:
(same goes for other stuff like the upgrade/new button presses above)